tests/integration/docusaurus/connecting_to_your_data/filesystem/spark_python_example.py

```diff
@@ -95,8 +95,36 @@
 assert isinstance(validator, ge.validator.validator.Validator)
 assert [ds["name"] for ds in context.list_datasources()] == ["my_filesystem_datasource"]
 assert (
-    "yellow_trip_data_sample_2018-01"
+    "yellow_trip_data_sample_2019-01"
     in context.get_available_data_asset_names()["my_filesystem_datasource"][
         "default_inferred_data_connector_name"
     ]
 )
+
+
+# Here is a RuntimeBatchRequest using a path to a directory
+batch_request = RuntimeBatchRequest(
+    datasource_name="my_filesystem_datasource",
+    data_connector_name="default_runtime_data_connector_name",
+    data_asset_name="example_data_asset",
+    runtime_parameters={"path": "taxi_data_files"},
+    batch_identifiers={"default_identifier_name": 1234567890},
+    batch_spec_passthrough={"reader_method": "csv", "reader_options": {"header": True}},
+)
+
+# Please note this override is only to provide good UX for docs and tests.
+# In normal usage you'd set your path directly in the BatchRequest above.
+batch_request.runtime_parameters["path"] = "data/"
+
+context.create_expectation_suite(
+    expectation_suite_name="test_suite", overwrite_existing=True
+)
+validator = context.get_validator(
+    batch_request=batch_request, expectation_suite_name="test_suite"
+)
+
+print(validator.head())
+print(validator.active_batch.data.dataframe.count())  # should be 30,000
```

Comment:
The primary intent of the `print()` statement here is to be displayed in the guide (the print statement not the output of the print statement), to show users how to interact with the `validator` object.  The purpose of the `assert` statement is for the integration testing (so it's not shown to the users). 

There might be better ways to do this, but this seemed like a simple way for a user to check "yes, this is the data that I expect to see".

Reply:
ah wait I actually really like that idea I totally forgot that we're linking these scripts in the docs so that users can run them on their own local machines.

Disregard my note your implementation looks perfect